include/assert.hpp

```diff
@@ -19,8 +19,7 @@
 
 #ifdef ASSERT_USE_MAGIC_ENUM
  // this is a temporary hack to make testing thing in compiler explorer quicker (it disallows simple relative includes)
- #include\
- "../third_party/magic_enum.hpp"
```

Comment:
Quick question about this change @qdewaghe, was this formatting causing issues with Cmake? I'd like to make things easier to play around with on https://godbolt.org and this was a workaround for how it handles relative includes

Reply:
Oh sorry I didn't put it back properly.

Reply:
Thanks for the quick reply! Yeah this one is a little weird :)

Reply:
I can make a PR later today to fix the cmake version, this and a few things friendlyanon was complaining about.

Reply:
Thanks so much for the cmake help!! You are a superstar!
I've gone ahead and quickly put this change back since I'm toying with CE now